doc/jvm/xgboost4j_spark_tutorial.rst

```diff
@@ -290,7 +290,20 @@ When interacting with other language bindings, XGBoost also supports saving-mode
 
 * You can build XGBoost4J-Spark with the steps described in `here <https://xgboost.readthedocs.io/en/latest/jvm/index.html#installation-from-source>`_, but turning `USE_HDFS <https://github.com/dmlc/xgboost/blob/e939192978a0c152ad7b49b744630e99d54cffa8/jvm-packages/create_jni.py#L18>`_ (or USE_S3, etc. in the same place) switch on. With this approach, you can reuse the above code example by replacing "nativeModelPath" with a HDFS path.
 
-However, if you build with USE_HDFS, etc. you have to ensure that the involved shared object file, e.g. libhdfs.so, is put in the LIBRARY_PATH of your cluster. To avoid the complicated cluster environment configuration, we recommend download file manually from HDFS and load with the pre-built version of XGBoost.
+However, if you build with USE_HDFS, etc. you have to ensure that the involved shared object file, e.g. libhdfs.so, is put in the LIBRARY_PATH of your cluster. To avoid the complicated cluster environment configuration, we recommend the following steps to pass models with HDFS, S3, etc. (taking HDFS as an example)
+
+* create a new file with **val outputStream = fs.create("hdfs_path")**
```

Comment:
Does `fs` here refer to an object of `org.apache.hadoop.fs.FileSystem`?

Reply:
I think it's OK to have this assumption here.....

Reply:
Let's add a sentence about using a binding of Hadoop then. Something like

... we recommend that you use the language-specific binding of Hadoop as follows:
```
private val fs = {
    val conf = new Configuration()
    FileSystem.get(conf)
  }
val outputStream = fs.create("hdfs_path")
```